### PR TITLE
Reconciliation in reports controller

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1363

### Change description ###
+ test

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by doncem — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java [hunk 1]
```diff
@@ -1,31 +1,23 @@
...
     public final String container;
-
-    @JsonProperty("type")
     public final String type;
-
-    @JsonProperty("stated")
     public final String stated;
-
-    @JsonProperty("actual")
     public final String actual;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java:8]: can you explain?

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ReportsControllerTest.java [hunk 2]
```diff
@@ -257,4 +273,39 @@ public void should_return_proper_response_when_there_are_no_rejected_files() thr
...
+                    .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                    .content(Resources.toString(getResource("reconciliation/envelopes.json"), UTF_8))
+            )
+            .andDo(print())
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(APPLICATION_JSON_VALUE))
+            .andExpect(content().json(Resources.toString(getResource("reconciliation/discrepancies.json"), UTF_8)));
+
+        // then
+        ArgumentCaptor<ReconciliationStatement> argument = ArgumentCaptor.forClass(ReconciliationStatement.class);
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ReportsControllerTest.java:305]: `var` can be used

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java (hunk 1 above)
Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java:8]: There is a new class DiscrepancyItem for Json presentation

**Review by rkondratowicz — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java (hunk 1 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java:8]: looks identical, but ok

**Pull request merged**